src/components/MultiHopTrade/components/TradeInput/components/ReceiveSummary.tsx

```diff
@@ -51,6 +43,7 @@ type ReceiveSummaryProps = {
   swapperName: string
   defaultIsOpen?: boolean
   swapSource?: SwapSource
+  priceImpact?: BigNumber
```

Comment:
`BigNumber.Value` to allow passing in strings and numbers instead of a BN?

Reply:
can do 4b0bc0addd07919bd08f9d6171516a9319e46ed7